src/workerd/server/tests/python/import_tests.bzl

```diff
@@ -32,11 +32,16 @@ def generate_wd_test_file(requirement):
     return WD_FILE_TEMPLATE.format(requirement, requirement)
 
 # to_test is a dictionary from library name to list of imports
-def gen_import_tests(to_test, pkg_skip_versions = {}):
+def gen_import_tests(to_test, python_version, pkg_skip_versions = {}):
     for lib in to_test.keys():
+        if lib.endswith("-tests"):
+            # TODO: Skip testing the import of `-tests` packages like `regex-tests`, they are failing
+            # for now and we need to fix them.
+            continue
```

Comment:
Not clear that we ever want to be testing this actually. I think that `pyodide-build-scripts` should be updated to not emit them.